app/models/package_manager/maven.rb

```diff
@@ -59,13 +59,19 @@ def self.recent_names
 
     def self.project(name)
       sections = name.split(':')
+      path = sections.join('/')
+      versions = versions({ path: path })
+      return {} unless versions.present?
```

Comment:
Are we nearly back to the original version of this method?

Reply:
🤷‍♂